gulpfile.js

```diff
@@ -523,17 +397,18 @@ export async function release() {
       await client.put(
         `/rest/api/3/issue/${encodeURIComponent(issueKey)}`,
         { update: { labels: [{ add: currentTag }] } },
-        { responseType: 'json', auth: { username: CLI_ARGS.jiraUser, password: CLI_ARGS.jiraApiKey } }
+        { responseType: 'json', auth: { username: cliArgs.jiraUser, password: cliArgs.jiraApiKey } }
       );
     } catch (error) {
       console.log(error);
     }
   }
 }
 
-export const up = gulp.series(mongoUp, minioUp, maildevUp);
+export const up = gulp.parallel(mongoUp, minioUp, maildevUp);
```

Comment:
Nice!

Reply:
Yeah, no clue why it was `series` before :grin: